BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppCoordinator.swift

```diff
@@ -138,29 +138,36 @@ final class AppCoordinator: Coordinator {
     }
     
     func processExternalDocument(withUrl url: URL, sourceApplication: String?) {
-        
+        var doc: GiniCaptureDocument?
         let captureConfiguration = configuration.captureConfiguration()
         // 1. Build the document
         let documentBuilder = GiniCaptureDocumentBuilder(documentSource: .appName(name: sourceApplication))
         documentBuilder.importMethod = .openWith
         
         documentBuilder.build(with: url) { [weak self] (document) in
             guard let self = self else { return }
-            
+            // Get saved imageData which was save in the extension from the group extension UserDefaults
+            // Build document with imageData because it's not possible to do it for the url from Photos
```

Comment:
[Suggestion] If it's not possible for the url from Photos, then I would recommend not calling `documentBuilder.build()` when coming from Photos. Hmm, maybe it would make sense to create a separate method like `processExternalDocumentFromPhotos()` which only deals with images coming from Photos and leave `processExternalDocument()` unchanged so that it only deals with "open with" files. #SingleResponsibilityPrinciple 😉